tiatoolbox/models/architecture/sccnn.py

```diff
@@ -10,12 +10,15 @@
 from __future__ import annotations
 
```

Comment:
I tried running sccnn on a WSI, and it errored because in line 1372 in semantic segmentor.py:

    total_size = np.max(matches).astype(np.uint16) + 1
    
uint16 wasnt big enough to represent the number. Changing to uint32 allowed it to run, but unusably slow. Is sccnn method just not suitable for WSI-scale usage? In which case, perhaps we should document this. Otherwise, if it is actually supposed to be useable on WSIs, then something may be broken that we need to look in to.